src/components/Navbar/MenuItems.tsx

```diff
@@ -125,20 +125,24 @@ export const menu = (onClickMenuItems?: VoidFunction) => {
         heading="Resources"
         dropdownItems={[
           <DropdownItem
+            key="docs"
```

Comment:
Fixes console warning about lack-of keys

Reply:
awesome